certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -553,6 +553,77 @@
         "x-ms-visibility": "advanced"
       }
     },
+    "/accounts/{accountId}/envelopes/{envelopeId}/notification": {
+      "put": {
+        "summary": "Add reminders for an envelope",
+        "description": "Add reminders for an envelope",
+        "operationId": "AddReminders",
+        "parameters": [
+          {
+            "name": "accountId",
+            "in": "path",
+            "description": "Account id",
+            "required": true,
+            "x-ms-summary": "Account",
+            "x-ms-test-value": "insert account id",
+            "x-ms-dynamic-values": {
+              "operationId": "GetLoginAccounts",
+              "value-collection": "loginAccounts",
+              "value-path": "accountIdGuid",
+              "value-title": "name"
+            },
+            "type": "string"
+          },
+          {
+            "name": "envelopeId",
+            "in": "path",
+            "description": "Envelope id",
+            "required": true,
+            "x-ms-summary": "Envelope",
+            "x-ms-test-value": "insert envelope id",
+            "type": "string"
+          },
+          {
+            "name": "reminderEnabled",
+            "in": "query",
+            "description": "When set to false the remiders are disabled. Default is false",
+            "required": true,
+            "x-ms-summary": "Reminder Enabled",
+            "x-ms-test-value": false,
+            "type": "boolean",
+            "default": false
+          },
+          {
+            "name": "reminderDelay",
+            "in": "query",
+            "description": "Add number of days after which recipient receives reminder email.",
+            "required": true,
+            "x-ms-summary": "Reminder Delay",
+            "type": "string",
```

Comment:
can this be a number ?

Reply:
number is not accepting default value as 0. So I declared it as string.